.github/workflows/codespell.yml

```diff
@@ -14,4 +14,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: codespell-project/actions-codespell@master
+      - name: Install dependencies
+        run: |
+          pip install codespell tomli
+      - name: Check spelling
+        run: |
+          codespell --config pyproject.toml
```

Comment:
I think you can just revert this. Their docs say `pyproject.toml` is the top priority (https://github.com/codespell-project/codespell#using-a-config-file) and https://github.com/codespell-project/actions-codespell/pull/61 installed toml support by default.